bip-process-revised.mediawiki

```diff
@@ -0,0 +1,423 @@
+<pre>
+  BIP: 3
```

Comment:
From the text: "(authors MUST NOT self-assign BIP numbers)". Set the example you want people to follow :)

Reply:
I think this is just a joke :) but just in case it isn't BIP editors assign BIP numbers. A BIP process revision needs a BIP editor to assign a number to it. If we were to be really pedantic Luke should assign the BIP number so Kalle isn't assigning his own BIP number. It seems clear though that 1-7 have been reserved for revised BIP processes and that a proposed revised BIP process to BIP 2 is obviously going to be BIP 3.

Reply:
I missed the initial comment. Yeah, I shouldn't have done that. Will address if I reopen this.